src/plugins/mdraid.c

```diff
@@ -17,11 +17,15 @@
  * Author: Vratislav Podzimek <vpodzime@redhat.com>
  */
 
+#define _XOPEN_SOURCE
```

Comment:
This is needed for the ``time.h`` include, I guess? Or something else? Either way, a comment with an explanation would be nice.